initials.c

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
```

Comment:
I would also comment your name and purpose of this code at the top for good practice.